src/js/dismiss.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * DISMISS
+ *
+ * @package uri-component-library
+ */
+
+( function() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function. (wrap-iife)](https://app.codacy.com/app/uriweb/uri-component-library/pullRequest?prid=4536390)